Dockerfile

```diff
@@ -1,7 +1,7 @@
 #
 # MIT License
 #
-# (C) Copyright 2020-2023 Hewlett Packard Enterprise Development LP
+# (C) Copyright 2020-2024 Hewlett Packard Enterprise Development LP
```

Comment:
It would be good if this also could have been reverted

Reply:
It came in part of the original PR, where i missed to remove it. Since the review went through and was approved, went ahead and had got the PR merged.

Will wait for @haasken-hpe review on the PR and proceed accordingly.

Reply:
I actually noticed it on the other PR and was going to comment and then I decided not to because  it might actually make our lives easier because whenever we make that temporary change to pull in an unstable build of cray-product-catalog or csm-api-client since we won't have to make the copyright year change just to make the license-check happy (at least for a few months).

But yeah, you're right generally, you don't want to make changes to the copyright year if you haven't otherwise updated the file.

Reply:
Thanks Ryan. That helps out, will proceed further.